lib/widgets/chat_list_welcome_tile.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:sloth/extensions/build_context.dart';
+import 'package:sloth/hooks/use_user_metadata.dart';
+import 'package:sloth/src/rust/api/welcomes.dart' show Welcome;
+import 'package:sloth/widgets/wn_animated_avatar.dart';
+
+class ChatListWelcomeTile extends HookWidget {
+  final Welcome welcome;
+  final VoidCallback? onTap;
+
+  const ChatListWelcomeTile({
+    super.key,
+    required this.welcome,
+    this.onTap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = context.colors;
+
+    final welcomerSnapshot = useUserMetadata(welcome.welcomer);
```

Comment:
🦥 This name es not very useful, I'd prefer something that makes clear that responds with the metadata